src/components/Reports/ReportFormAnnual.tsx

```diff
@@ -1,36 +1,389 @@
-import React from 'react';
-import { Box, Typography, useTheme } from '@mui/material';
-import { Textfield } from '@terraware/web-components';
+import React, { useState } from 'react';
+import { Grid, Typography, useTheme } from '@mui/material';
+import { Button, Checkbox, Textfield } from '@terraware/web-components';
 import { Report } from 'src/types/Report';
+import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import useDebounce from 'src/utils/useDebounce';
+import { SustainableDevelopmentGoal, SDG } from 'src/types/Report';
+import useSDGProgress from 'src/components/Reports/useSDGProgress';
+
+const DEBOUNCE_TIME_MS = 500;
 
 export type ReportFormAnnualProps = {
-  editable?: boolean;
+  editable: boolean;
   report: Report;
-  onChange?: (report: Report) => void;
+  updateDetails?: (field: string, value: any) => void;
+  updateSDGProgress?: (index: number, value: string) => void;
 };
 
 export default function ReportFormAnnual(props: ReportFormAnnualProps): JSX.Element {
-  const { editable, report } = props;
+  const { editable, report, updateDetails, updateSDGProgress } = props;
   const theme = useTheme();
+  const { isMobile, isTablet } = useDeviceInfo();
+
+  const MONTHS = [
+    strings.MONTH_01,
+    strings.MONTH_02,
+    strings.MONTH_03,
+    strings.MONTH_04,
+    strings.MONTH_05,
+    strings.MONTH_06,
+    strings.MONTH_07,
+    strings.MONTH_08,
+    strings.MONTH_09,
+    strings.MONTH_10,
+    strings.MONTH_11,
+    strings.MONTH_12,
+  ];
+
+  const SDG_STRING: { [key in SustainableDevelopmentGoal]: string } = {
+    NoPoverty: strings.SDG_NO_POVERTY,
+    ZeroHunger: strings.SDG_ZERO_HUNGER,
+    GoodHealth: strings.SDG_GOOD_HEALTH_AND_WELL_BEING,
+    QualityEducation: strings.SDG_QUALITY_EDUCATION,
+    GenderEquality: strings.SDG_GENDER_EQUALITY,
+    CleanWater: strings.SDG_CLEAN_WATER_AND_SANITATION,
+    AffordableEnergy: strings.SDG_AFFORDABLE_AND_CLEAN_ENERGY,
+    DecentWork: strings.SDG_DECENT_WORK_AND_ECONOMIC_GROWTH,
+    Industry: strings.SDG_INDUSTRY_INNOVATION_AND_INFRASTRUCTURE,
+    ReducedInequalities: strings.SDG_REDUCED_INEQUALITIES,
+    SustainableCities: strings.SDG_SUSTAINABLE_CITIES_AND_COMMUNITIES,
+    ResponsibleConsumption: strings.SDG_RESPONSIBLE_CONSUMPTION_AND_PRODUCTION,
+    ClimateAction: strings.SDG_CLIMATE_ACTION,
+    LifeBelowWater: strings.SDG_LIFE_BELOW_WATER,
+    LifeOnLand: strings.SDG_LIFE_ON_LAND,
+    Peace: strings.SDG_PEACE_JUSTICE_AND_STRONG_INSTITUTIONS,
+    Partnerships: strings.SDG_PARTNERSHIPS_FOR_THE_GOALS,
+  };
+
+  const [projectSummary, setProjectSummary] = useState(report.annualDetails?.projectSummary ?? '');
+  useDebounce(projectSummary, DEBOUNCE_TIME_MS, (value) => {
```

Comment:
just curious.. why debounce? is it too slow to edit otherwise?

Reply:
I did it for performance reasons. I didn't want to call updates to the report every keystroke. I thought it would be better to update state locally and debounce those updates to full report state. But honestly I didn't test without debounce, so it could be a premature optimization.

Reply:
i figured..  my thought is it's worth testing.. since user will be typing in one input at a time. should be ok to call onChange for every key stroke. 
However,, since we are updating the parent report object.. not sure if that change is triggering a lot of re-renders across the report.

Reply:
My sense is that it would cause a lot of rerenders. If `updateDetails` is called each keystroke, it updates the entire report object, which is a dependency of every item in the form. I'll test and see what it's like without the debounce though!

Reply:
Actually, it is smart enough not to repaint everything besides the input that is being interacted with. Thus, this seems to work fine without the debounce. So my intuition was wrong here and I can change this to get rid of the debounces 😄

Reply:
Thanks for confirming!!! Sounds good to me :)  I think this can be done as a follow up if you want?

Reply:
I can get it done pretty quickly for the annual details and follow up for the other part of the report.

Reply:
Looks good 🙌